packages/core/src/odata/v4/request-builder/request/uri-conversion/get-expansion.ts

```diff
@@ -0,0 +1,58 @@
+/* Copyright (c) 2020 SAP SE or an SAP affiliate company. All rights reserved. */
+
+import { Entity } from '../../../entity';
```

Comment:
Where does this file come from and is it necessary?

Reply:
hm, no idea. Deleted